subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -647,9 +766,24 @@ static void rp_cu_st_wait_rx_conn_update_ind(struct ll_conn *conn, struct proc_c
 {
 	switch (evt) {
 	case RP_CU_EVT_CONN_UPDATE_IND:
-		pdu_decode_conn_update_ind(ctx, param);
-		ctx->state = RP_CU_STATE_WAIT_INSTANT;
-		break;
+		switch(conn->lll.role) {
+		case BT_HCI_ROLE_MASTER:
+			ctx->unknown_response.type = PDU_DATA_LLCTRL_TYPE_CONN_UPDATE_IND;
+			rp_cu_send_unknown_rsp(conn, ctx, evt, param);
+			break;
+		case BT_HCI_ROLE_SLAVE:
+			pdu_decode_conn_update_ind(ctx, param);
+			//TODO(tosk): skip/terminate if instant passed?
+#if defined(CONFIG_BT_CTLR_CONN_PARAM_REQ)
+			/* conn param req procedure, if any, is complete */
+			conn->procedure_expire = 0U;
```

Comment:
void ull_conn_prt_clear(struct ll_conn *conn);

Should probably be changed to an inline, however it still does the thing;-)

Reply:
Done